src/microbe_stage/Microbe.tscn

```diff
@@ -14,6 +14,7 @@ render_priority = 1
 shader = ExtResource( 4 )
 shader_param/wigglyNess = 1.0
 shader_param/movementWigglyNess = 1.0
+shader_param/dissolveValue = null
```

Comment:
Does this affect something?
If godot wants to apply this change automatically, we should accept this after testing that nothing breaks.

Reply:
Maybe? I'm not actually getting that one on my end.

Reply:
This change? It doesn't happen automatically?
If it doesn't then I'd like to see this change being rolled back just in case it affects something.

Reply:
Oh, it did get added automatically. I actually didn't even notice it was in the commit till you just pointed it out. I'm not entirely sure what effect it has, if any.

Reply:
It's likely related to the death shader on the cells. It's probably fine. @Kasterisk can you check if this causes any issues?
I'm assuming it would cause noticeable issues in just a few minutes of playtesting, if it did cause issues.

Reply:
It shouldn't cause any issues, it's there so the shader's uniform variable `dissolveValue` could be changed on the inspector if I'm not mistaken